administrator/language/en-GB/plg_schemaorg_book.ini

```
@@ -0,0 +1,32 @@
+; Joomla! Project
+; (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_SCHEMAORG_BOOK="Schema.org - Book"
+PLG_SCHEMAORG_BOOK_DESCRIPTION_LABEL="A Schema.org Type <a href=\"https://schema.org/Book\" target=\"_blank\">More information</a>"
+PLG_SCHEMAORG_BOOK_FIELD_ABRIDGED_LABEL="Abridged"
+PLG_SCHEMAORG_BOOK_FIELD_ADDRESS_LABEL="Address"
+PLG_SCHEMAORG_BOOK_FIELD_AUDIOBOOK_LABEL="Audiobook"
+PLG_SCHEMAORG_BOOK_FIELD_AUTHOR_LABEL="Author"
+PLG_SCHEMAORG_BOOK_FIELD_BOOK_EDITION_LABEL="Book Edition"
+PLG_SCHEMAORG_BOOK_FIELD_BOOK_FORMAT_LABEL="Book Format"
+PLG_SCHEMAORG_BOOK_FIELD_DATE_PUBLISHED_LABEL="Date Published"
+PLG_SCHEMAORG_BOOK_FIELD_EMAIL_LABEL="Email"
+PLG_SCHEMAORG_BOOK_FIELD_E_BOOK_LABEL="E Book"
+PLG_SCHEMAORG_BOOK_FIELD_GENERIC_FIELD_LABEL="Generic Field"
+PLG_SCHEMAORG_BOOK_FIELD_GENERIC_TITLE_LABEL="Title"
+PLG_SCHEMAORG_BOOK_FIELD_GENERIC_VALUE_LABEL="Value"
+PLG_SCHEMAORG_BOOK_FIELD_GRAPHIC_NOVEL="Graphic Novel"
+PLG_SCHEMAORG_BOOK_FIELD_HARDCOVER_LABEL="Hardcover"
+PLG_SCHEMAORG_BOOK_FIELD_ILLUSTRATOR_LABEL="Illustrator"
+PLG_SCHEMAORG_BOOK_FIELD_ISBN_LABEL="ISBN"
+PLG_SCHEMAORG_BOOK_FIELD_LOCALITY_LABEL="Locality"
+PLG_SCHEMAORG_BOOK_FIELD_NAME_LABEL="Name"
+PLG_SCHEMAORG_BOOK_FIELD_NAME_LABEL="Title"
+PLG_SCHEMAORG_BOOK_FIELD_NUMBER_OF_PAGES_LABEL="Number of pages"

```

```suggestion
PLG_SCHEMAORG_BOOK_FIELD_NUMBER_OF_PAGES_LABEL="Number of Pages"

```

this is our styleguide